### ISSUE TITLE
node-red-node-tail information is old and has a wrong default value

### ISSUE THREAD

**Issue by sakazuki** [node-red/node-red-nodes#555]
### Which node are you reporting an issue on?

- node-red-node-tail

### What are the steps to reproduce?

- Show information tab
- Edit tail node

### What happens?

- No information about windows support
- Default regexp is wrong

![image](https://user-images.githubusercontent.com/1878694/61089116-157aa980-a475-11e9-8829-b09e6243c5b3.png)

### What do you expect to happen?

- Update information about windows support [5b34702]
   https://github.com/node-red/node-red-nodes/blob/master/storage/tail/28-tail.html#L28-L29

- Escape character in html file
  https://github.com/node-red/node-red-nodes/blob/master/storage/tail/28-tail.html#L43

```javascript
-           split: {value:"[\r]{0,1}\n"},
+           split: {value:"[\\r]{0,1}\\n"},
```

### Please tell us about your environment:

- [x] Node-RED version: 0.20.6
- [x] node.js version: 12.0.0
- [x] npm version: 6.10.0
- [x] Platform/OS: Windows
- [x] Browser: chrome

**Comment by dceejay** [comment]
Have pushed a change to help visualise the regex. Also have updated the English info... Would appreciate a Ja (and ideally Chinese) translation for - 

    <p>Note: On Windows you may need to close the file between writes for any updates to be registered.</p>

**Comment by sakazuki** [comment]
Thank you for your updates.

Japanese translation candidate is 

```
<p>注: Windowsでは、追加されたデータを出力するには、書き込みと書き込みの間にファイルを閉じる必要があります。</p>
```

**Comment by dceejay** [comment]
closed by  55698e637d0ca04ac812b7cf2284c5035ca7f010
and f7fa242d4d4536bcf2dcc7821b2d2d03f012a634
